aws/sdk/integration-tests/s3/tests/ignore-invalid-xml-body-root.rs

```diff
@@ -76,5 +77,5 @@ async fn ignore_invalid_xml_body_root() {
         .await
         .unwrap();
 
-    conn.assert_requests_match(&[]);
+    conn.assert_requests_match(&[AUTHORIZATION]);
```

Comment:
Why must we ignore `AUTHORIZATION` here? Does EP 2.0 mean we can't assume that header is deterministic?

Reply:
no, we just weren't testing the auth header here and I didn't want to both fixing it